<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RepeatedEventWidget</class>
 <widget class="QWidget" name="RepeatedEventWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>187</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="titleLabel">
         <property name="text">
          <string>title</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="repearEveryLabel">
         <property name="text">
          <string>repear every</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="enabledLabel">
         <property name="text">
          <string>enabled</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="contentLabel">
       <property name="text">
        <string>description</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="enableButton">
       <property name="text">
        <string>Enable</string>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::AppointmentNew"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="disableButton">
       <property name="text">
        <string>Disable</string>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::ListRemove"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="editButton">
       <property name="text">
        <string>Edit</string>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::DocumentProperties"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteButton">
       <property name="text">
        <string>Delete</string>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::EditDelete"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
